### PR TITLE
[MRG] Fixes to example and website + constant.

### PR DESCRIPTION
**Pull request by jaeilepp**
Closes https://github.com/mne-tools/mne-python/issues/3025
The mayavi figure problem seems to have fixed itself. See https://circle-artifacts.com/gh/mne-tools/mne-python/501/artifacts/1/home/ubuntu/mne-python/doc/_build/html/auto_examples/index.html
Now trying to replicate the success for stable 0.11.

### CODE REVIEW

**Comment by agramfort** [comment]
Great 

Lgtm

File: examples/realtime/plot_compute_rt_average.py [hunk 1]
```diff
@@ -51,11 +51,11 @@
 rt_client.send_data(rt_epochs, picks, tmin=0, tmax=150, buffer_size=1000)
 for ii, ev in enumerate(rt_epochs.iter_evoked()):
     print("Just got epoch %d" % (ii + 1))
+    ev.pick_types(meg=True, eog=False)  # leave out the eog channel
     if ii == 0:
         evoked = ev
     else:
-        evoked += ev
```
Comment (agramfort) [inline examples/realtime/plot_compute_rt_average.py]: I think Its a bug

**Comment by Eric89GXL** [comment]
> The mayavi figure problem seems to have fixed itself.

It didn't occur to me at the time, but it's probably due to the `offscreen` line here:

https://github.com/mne-tools/mne-python/pull/3024/files#diff-684a0ceb2878d48b19612ca360e277efR271

**Comment by Eric89GXL** [comment]
Thanks @jaeilepp

**Pull request merged**